README.md

```diff
@@ -2,3 +2,111 @@
 
 [![CircleCI](https://circleci.com/gh/thoughtbot/middleman-inline_svg.svg?style=svg)](https://circleci.com/gh/thoughtbot/middleman-inline_svg)
 
+A [Middleman] extension that enables the inlining of SVG's. This gives us the ability to style those SVG's using standard CSS.
+
+## Installation
+
+1. Add middleman-inline_svg to your `Gemfile`:
+
+```ruby
```

Comment:
I believe these code blocks need to be indented with 4 spaces to render the list properly.